Source/GoBot/ViewDatabase.swift

```diff
@@ -924,7 +924,7 @@ class ViewDatabase {
             .join(self.abouts, on: self.abouts[colAboutID] == self.msgs[colAuthorID])
             .join(.leftOuter, self.posts, on: self.posts[colMessageRef] == self.tangles[colMessageRef])
             .join(.leftOuter, self.votes, on: self.votes[colMessageRef] == self.tangles[colMessageRef])
-            .filter(colMsgType == ContentType.post.rawValue)
+            .filter(colMsgType == ContentType.post.rawValue || colMsgType == ContentType.vote.rawValue )
```

Comment:
Maybe you want to consider to update addNumberOfPeopleReplied as well to take into account the number of people that replied with a like. And I couldn't see likes, probably we are filtering them in other places?

I could show votes once, but I felt that we should add design there. Just displaying a thumbs up with the avatar, name, date, and options button to follow/unfollow/etc seems to me many UI distracting elements. Maybe we can come with a better design that either show votes as individual comments, or a total number of likes somewhere.

Reply:
But I would do this in a different branch, not this one.

Reply:
it does show votes as just single emoji replies at this point. I agree something more compact and letting you click through to see who reacted is better.